public_html/wp-content/mu-plugins/blocks/assets/src/shared/featured-image/inspector-control.js

```diff
@@ -2,48 +2,33 @@
  * WordPress dependencies.
  */
 const { Component } = wp.element;
-const { PanelBody, PanelRow, TextControl, BaseControl, SelectControl } = wp.components;
+const { PanelBody, PanelRow, TextControl, BaseControl, SelectControl, ToggleControl } = wp.components;
 const { __ } = wp.i18n;
+import './style.scss';
 
 /**
  * Not sure if these sizes are actually useful.
```

Comment:
`[SHOULD]` We should be sure before merging this, and update the comment to describe _what_ the var is (or just remove if unnecessary).

Reply:
Added as part of https://github.com/WordPress/wordcamp.org/issues/36